californium-core/src/main/java/org/eclipse/californium/core/network/InMemoryMessageExchangeStore.java

```diff
@@ -116,7 +116,7 @@ public InMemoryMessageExchangeStore(final NetworkConfig config, TokenGenerator t
 
 	private void startStatusLogging() {
 
-		final int healthStatusInterval = config.getInt(NetworkConfig.Keys.HEALTH_STATUS_INTERVAL, 60); // seconds
+		final int healthStatusInterval = config.getInt(NetworkConfig.Keys.HEALTH_STATUS_INTERVAL, 0); // seconds
```

Comment:
I would prefer to add the `0` to `NetworkConfigDefaults` as field and use then that field in all places, where the default is intended.